iocage_lib/ioc_create.py

```diff
@@ -326,17 +350,20 @@ def _create_jail(self, jail_uuid, location):
 
         # This test is to avoid the same warnings during install_packages.
 
-        if jail_uuid == "default" or jail_uuid == "help":
-            iocage_lib.ioc_destroy.IOCDestroy(
-            ).__destroy_parse_datasets__(
-                f"{self.pool}/iocage/jails/{jail_uuid}")
-            iocage_lib.ioc_common.logit({
-                "level": "EXCEPTION",
-                "message": f"You cannot name a jail {jail_uuid}, "
-                           "that is a reserved name."
-            },
-                _callback=self.callback,
-                silent=self.silent)
+        if not self.plugin:
```

Comment:
Why this check? The jail names are reserved for plugins or jails, that doesn't matter.